### PR TITLE
Link to license list

### PR DESCRIPTION
**Pull request by raboof**
Save users a trip to the search engine

### CODE REVIEW

**Review by mxmehl — APPROVED** [review]
Good idea, thank you.

File: src/reuse/init.py [hunk 1]
```diff
@@ -23,7 +23,8 @@ def prompt_licenses(out: IO[str] = sys.stdout) -> List[str]:
     """Prompt the user for a list of licenses."""
     first = _(
         "What license is your project under? "
-        "Provide the SPDX License Identifier."
+        "Provide the SPDX License Identifier. "
+        "See https://spdx.org/licenses/ for the list."
```
Comment (carmenbianca) [inline src/reuse/init.py:27]:
```suggestion
        "See <https://spdx.org/licenses/> for the list."
```

The added brackets should make it easier to click in some terminals.

Comment (raboof) [reply src/reuse/init.py]: Updated and checked that while the terminal I use (kitty) doesn't need this, it also isn't thrown off by it, so :+1: by me

**Review by carmenbianca — APPROVED** [review]
Awesome. Thanks a lot for this contribution @raboof

**Pull request merged**